dashboard/app/controllers/transfers_controller.rb

```diff
@@ -89,6 +89,13 @@ def create
 
     if new_section.user != current_user
       new_section_teacher = new_section.user
+      if new_section_teacher.nil?
```

Comment:
After https://github.com/code-dot-org/code-dot-org/pull/13464, you could instead check `new_section.invalid?`